frontend/src/app/components/Header.js

```diff
@@ -38,6 +38,10 @@ export default class Header extends React.Component {
     return this.props.hasAddon;
   }
 
+  setHeaderLinkPath() {
```

Comment:
because of the design of the new loader, i noticed a little double-tap when the user clicks on the TXP wordmark if the add-on is installed. This happens b/c the site has to step through a redirect which requires the react state to load before it starts. Adding this method to filter the URL pre-click should make the experience snappier